docs/src/main/asciidoc/grpc-xds.adoc

```diff
@@ -0,0 +1,83 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Using xDS gRPC
```

Comment:
This heading will make Vale very unhappy :-D 
We need to add `xDS` and `gRPC` to the excluded word list (@michelle-purcell / @ebullient )

Reply:
Fixed in PR #30256.